classes/player/player.gd

```diff
@@ -287,8 +287,10 @@ func player_physics():
 	manage_invuln()
 	manage_buffers()
 	manage_dive_recover()
-	manage_triple_flip()
-	manage_backflip_flip()
+	if state == S.TRIPLE_JUMP:
```

Comment:
In future it might be worthwhile taking code for stuff like this and moving it into child nodes to form a more state-machine-like approach, but this is fine for now.

Reply:
Oh yeah, that'd definitely be better. This whole script is a huge mess 😆